public/index.html

```diff
@@ -93,148 +93,169 @@
+                        <div class="col-lg-3 col-centered thumb">
+                            <a class="thumbnail" href="#">
+                                <img id="nebula-21" class="img-responsive" src="" alt="">
+                            </a>
+                        </div>
                     </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-11" class="img-responsive" src="" alt="">
-                        </a>
+                    <div class="row-centered">
+                        <div class="col-lg-3 col-centered thumb">
+                            <a class="thumbnail" href="#">
+                                <img id="nebula-22" class="img-responsive" src="" alt="">
+                            </a>
+                        </div>
+                        <div class="col-lg-3 col-centered thumb">
+                            <a class="thumbnail" href="#">
+                                <img id="nebula-23" class="img-responsive" src="" alt="">
+                            </a>
+                        </div>
+                        <div class="col-lg-3 col-centered thumb">
+                            <a class="thumbnail" href="#">
+                                <img id="nebula-24" class="img-responsive" src="" alt="">
+                            </a>
+                        </div>
                     </div>
                 </div>
-                <div class="row-centered">
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-16" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-15" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-14" class="img-responsive" src="" alt="">
-                        </a>
-                    </div>
-                    <div class="col-lg-3 col-centered thumb">
-                        <a class="thumbnail" href="#">
-                            <img id="nebula-13" class="img-responsive" src="" alt="">
-                        </a>
+            </div>
+        </div>
+        <!-- CLUSTER COSMOLOGIST -->
+        <div class="tab-pane fade" id ="clusterplot">
+            <div class="row">
+                <div class="row">
+                    <div class="col-lg-4 plotbox">
+                        <label for="cluster-slider">Number Clusters:</label>
+                        <input id="cluster-slider" type="range" min="2" max="20" value="2" step="1"/>
```

Comment:
Put this in the intermediate value, ie 10? 2 looks really low for most cases, and makes the display look a bit broken.